# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../icecream/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur icecream-orig/mon/hostinfo.h icecream/mon/hostinfo.h
--- icecream-orig/mon/hostinfo.h	2004-08-27 11:24:46.000000000 +0000
+++ icecream/mon/hostinfo.h	2006-04-23 09:16:20.266846500 +0000
@@ -54,7 +54,7 @@
   protected:
     static void initColor( const QString &value, const QString &name );
 
-    QColor HostInfo::createColor();
+    QColor createColor();
     QColor createColor( const QString &name );
 
   private:
diff -Nur icecream-orig/mon/starview.h icecream/mon/starview.h
--- icecream-orig/mon/starview.h	2005-01-13 11:56:26.000000000 +0000
+++ icecream/mon/starview.h	2006-04-23 09:18:56.576615250 +0000
@@ -142,7 +142,7 @@
 
     QString id() const { return "star"; }
 
-    HostItem *StarView::findHostItem( unsigned int hostid );
+    HostItem *findHostItem( unsigned int hostid );
 
     void checkNode( unsigned int hostid );
 
